packer/ansible/roles/localsource/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: install dev libraries for NPM installs
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - libkrb5-dev
+    - unzip
+  sudo: yes
+
+- name: Install RackHD from local source
+  copy: src="../../{{ item }}" dest="{{ ansible_env.HOME }}/src/"
```

Comment:
That is the base dir of RackHD as referenced from the directory of the playbook.